integration-cli/docker_api_containers_test.go

```diff
@@ -1971,7 +1971,8 @@ func (s *DockerAPISuite) TestContainersAPICreateMountsCreate(c *testing.T) {
 			assert.NilError(c, err)
 			defer os.RemoveAll(tmpDir3)
 
-			assert.Assert(c, mountWrapper(tmpDir3, tmpDir3, "none", "bind,shared") == nil)
+			err = mountWrapper(c, tmpDir3, tmpDir3, "none", "bind,shared")
+			assert.Check(c, err)
 
 			cases = append(cases, []testCase{
 				{
```

Comment:
Do we want to append these test cases (at least the last one) only if this check succeeds?

```suggestion
			if assert.Check(c, mountWrapper(c, tmpDir3, tmpDir3, "none", "bind,shared")) {

			    cases = append(cases, []testCase{
				    {
```

Otherwise, do we even need this check at all? Maybe we can just remove it?

Reply:
Good point; I rebased, and applied your suggestion